dxr/plugins/js/analyze_js/analyze_tree.js

```diff
@@ -0,0 +1,81 @@
+const path = require("path");
+const fs = require('fs');
```

Comment:
Eventually we should standardize on a kind of quotes. And use eslint. I have a good default config in Fathom.